flink-table/flink-table-common/src/test/java/org/apache/flink/table/catalog/CatalogTest.java

```diff
@@ -1069,6 +1070,84 @@ public void testListPartitionPartialSpec() throws Exception {
 		assertEquals(1, catalog.listPartitions(path1, createAnotherPartitionSpecSubset()).size());
 	}
 
+
+	// ------ table and column stats ------
+
+	@Test
+	public void testGetTableStats_TableNotExistException() throws Exception{
+		catalog.createDatabase(db1, createDb(), false);
+		exception.expect(org.apache.flink.table.catalog.exceptions.TableNotExistException.class);
+		catalog.getTableStatistics(path1);
+	}
+
+	@Test
+	public void testGetPartitionStats() throws Exception{
+		catalog.createDatabase(db1, createDb(), false);
+		catalog.createTable(path1, createPartitionedTable(), false);
+		catalog.createPartition(path1, createPartitionSpec(), createPartition(), false);
+		CatalogTableStatistics tableStatistics = catalog.getPartitionStatistics(path1, createPartitionSpec());
+		assertEquals(0, tableStatistics.getFileCount());
```

Comment:
minor: use `HiveStatsUtil.DEFAULT_STATS_ZERO_CONST` as expected value?

Reply:
it will cause an unwanted reverse dependency for HiveStatsUtil is a class of flink-connector-hive.